tests/networkpolicy_test.go

```diff
@@ -12,35 +15,25 @@ import (
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
 )
 
+func pingEventually(fromVmi *v1.VirtualMachineInstance, toIp string) AsyncAssertion {
+	return Eventually(func() error {
+		By(fmt.Sprintf("Pinging from VMI %s/%s to Ip %s", fromVmi.Namespace, fromVmi.Name, toIp))
+		return tests.PingFromVMConsole(fromVmi, toIp)
```

Comment:
yes as long as we don't change behavior. ping should have a timeout.

Reply:
we wont change it so fast now that we have the new mechanism, there is timeout in the mechanism itself, all the code use this, those arguments were there when it didnt use the mechanism

Reply:
The helper has a default timeout of 5sec (-w) and an expected timeout of 10sec.
By removing the `-w 3` you added 2sec to this test in case it fails.

If it is a matter of retry you can just drop the `-c` and there is no need for the eventually.

Reply:
as we spoke we need the eventually at least on negative tests, because the first ping might success, until the policy is effective and blocking the traffic.
a successful ping would return immediately, and wont consume even 3 or 5 seconds.

on the positive tests, i will check what can be done, in case its important.